apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -239,6 +245,14 @@ function InvoiceItem({
 }) {
   const t = useScopedI18n("invoicegenerator");
 
+  const [quantity, setQuantity] = useState(0);
+  const [unitPrice, setUnitPrice] = useState(0);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  useEffect(() => {
```

Comment:
This now causes unnecessary renders when it's a separate state and updated via `useEffect`. Instead we could probably be calculating it directly on the fly from `quantity * unitPrice`